docroot/.htaccess

```diff
@@ -69,9 +69,10 @@ AddEncoding gzip svgz
 <IfModule mod_rewrite.c>
   RewriteEngine on
 
-  # Forbid access to admin and node paths on the api2.foia.gov domain.
+  # Forbid access to admin, node, and user paths on domains other than admin.foia.gov and local.dojfoia.gov.
+  RewriteCond %{HTTP_HOST} !local.dojfoia.gov
```

Comment:
It would be nice if our local environment was setup to match the admin.foia.gov pattern, rather than having to make an exception here. Let me look into whether or not it's reasonable to change that at this stage.